lib/Myriad/Service/Attributes.pm

```diff
@@ -87,6 +87,8 @@ This will cause the method to be registered in L<Myriad::Registry/add_rpc>.
 
 sub rpc {
     my ($class, $pkg, $method, $code, $args) = @_;
+    # Set default $args if not passed
+    $args = { process_pending => 0 } unless $args and exists $args->{process_pending};
```

Comment:
This is a bit of nitpicking, but `process_pending` is a bit of Redis terminology, maybe another one-word term?